app/page.tsx

```diff
@@ -95,7 +94,7 @@ export default async function Dashboard() {
         <h4 className="scroll-m-20 text-3xl font-semibold text-center py-4">
           Skylight
         </h4>
-
+        <a className="scroll-m-20 mx-20 text-m" href="/api/auth/logout">Logout</a>
```

Comment:
Please put this down at the bottom of the page somewhere, and in a smaller font.

Reply:
Sure!

Reply:
I've thrown the button down at the bottom for now, we might want to do a design pass on that eventually. (The logout button is also not strictly necessary because the SSO should just expire - so maybe we could just drop it?)